src/components/common/Stepper.vue

```diff
@@ -52,6 +72,24 @@ export default class Stepper extends Vue {
   justify-content: center;
 }
 
+.step:hover {
+  cursor: pointer;
+
+  .step__btn {
+    background-color: #1976d2;
+    color: $BCgovInputBG;
```

Comment:
There's something not quite right here. If you select the Incorporation Agreement icon (handshake), you will notice that the icon has a square around it in a different hue than the fab button.

Reply:
hmmm.. I'll look into this!

Reply:
Updated in next commit!

Reply:
Almost! looks like the check mark icon may also need a -1 tabIndex, when I try to tab through any of the steps that are valid it takes 2 tabs.